galgebra/lt.py

```diff
@@ -12,12 +12,24 @@
 from sympy import (
     expand, symbols, Matrix, Transpose, zeros, Symbol, Function, S, Add, Expr
 )
+from sympy.printing.latex import LatexPrinter as _LatexPrinter
```

Comment:
In future, I wish to use `__all__` to control the exposure instead of prefix `_`

Reply:
I'm a little torn about that. I've seen no evidence that anyone actually looks at `__all__` to decide whether to consider a member private or not. But then again, we import so much junk from sympy, it seems hardly worth using the `_` prefix here.

Reply:
Yeah, never mind.

Reply:
Something to cleanup in future I think - at any rate, it would be good to add `__all__`.